beginner_source/scaled_dot_product_attention_tutorial.py

```diff
@@ -0,0 +1,320 @@
+"""
+An overview of torch.nn.functional.scaled_dot_product_attention
+===============================================================
+
+"""
+
+
+######################################################################
+# Summary
+# ~~~~~~~~
+# 
+# In this tutorial we want to highlight a new ``torch.nn.functional`` function
+# that can be helpful for implementing transformer architectures. The
+# function is named ``torch.nn.functional.scaled_dot_product_attention``.
+# There is some extensive documentation on the function in the `PyTorch
+# documentation <https://pytorch.org/docs/master/generated/torch.nn.functional.scaled_dot_product_attention.html#torch.nn.functional.scaled_dot_product_attention>`__.
+# This function has already been incorporated into torch.nn.MHA
```

Comment:
```suggestion
# This function has already been incorporated into torch.nn.MultiheadAttention
```

(since `torch.nn.MHA` doesn't exist, I suggest using the full name to avoid confusion)